src/main/java/de/focusshift/zeiterfassung/feedback/FeedbackViewController.java

```diff
@@ -2,13 +2,17 @@
 
 import de.focusshift.zeiterfassung.tenancy.user.EMailAddress;
 import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.security.access.prepost.PreAuthorize;
 import org.springframework.security.core.annotation.AuthenticationPrincipal;
 import org.springframework.security.oauth2.core.oidc.user.OidcUser;
+import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.servlet.mvc.support.RedirectAttributes;
 
+@Controller
```

Comment:
The Controller is published as bean from the `FeedbackConfiguration` if enabled. So there is no need for a @Controller annotation